content/03.methods.md

```diff
@@ -106,10 +106,23 @@ Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lan
 
 We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.
 
+#### Consensus SNV Calling
+
+We considered mutations to be the same across callers if they were identical for the following MAF fields: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
+We compared all four callers in their overlap with each other, variant allele frequencies of their calls, base changes called, and genomic regions of their calls [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/compare_snv_callers_plots.Rmd].
+VarDict called numerous variants with low allele frequency [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/vaf_violin_plot.png] in addition to nearly all of the calls identified by the other three callers [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/upset_plot.png].
+Because our filtered set was based on the intersection, we removed it from the consensus calculation.
+Strelka2 does not call multinucleotide variants (MNV), but instead calls each component SNV as a separate mutation, so we separated MNV calls from Mutect2 and Lancet into consecutive SNVs before comparing them with Strelka2.
```

Comment:
I'm confused about why we're doing multinucleotide variants in the SNV section.

Reply:
The data are all mixed in and different callers call consecutive SNVs differently. I don't know where else this information should go but I'm open to suggestions if there's a better place to put it. I think it is something people should be aware of so I don't necessarily think we should drop it entirely.

Reply:
We have effectively reduced MNVs to consecutive SNVs in the consensus, so I feel like this is the appropriate place to put it. MNVs are distinct from copy number and structural variants, so we won't want to put them in any other place, and I don't think they deserve there own section.

Reply:
On second look, I think it makes sense to move this sentence up to after line 111 ("Our SNV calling process led to..."